pkg/koordlet/qosmanager/plugins/cpuburst/cpu_burst.go

```diff
@@ -355,6 +355,11 @@ func (b *cpuBurst) applyCFSQuotaBurst(burstCfg *slov1alpha1.CPUBurstConfig, podM
 			continue
 		}
 
+		if containerStat.State.Running == nil {
+			klog.Infof("skip container %s/%s/%s, because it is not running", pod.Namespace, pod.Name, containerStat.Name)
```

Comment:
how about `klog.V(6).Infof` for skip message

Reply:
> how about `klog.V(6).Infof` for skip message

ok, I have resolved it.